runtime/util/ConfigurationPrinter.cpp

```diff
@@ -284,3 +288,89 @@ void printSortedConfigurationToFile(
   writer w = {file, nullptr};
   printConfigurationInternal(&w, subject, sort, false, &state);
 }
+
+void printMatchResult(
+    std::ostream &os, MatchLog *matchLog, size_t logSize,
+    std::string &definitionPath) {
+  char subjectFilename[15] = "subject_XXXXXX";
+  int sf = mkstemp(subjectFilename);
+  FILE *subject = fdopen(sf, "w");
```

Comment:
Not the right place to fix this as we do the same thing in several other places in the backend, but see: https://github.com/runtimeverification/llvm-backend/issues/738

Reply:
Any ideas on the best way to handle this? I may change here and take the PR to work on applying the same solution!

Reply:
Sure, I'll write a bit more context in #738 and assign you.